atdumpcardsv2.py

```diff
@@ -166,7 +166,15 @@ def list_v2_cards(field_id, project_id, column_name, file_name=None):
                         break
                 if (fieldValue.get('field') != None and (fieldValue['field']['name'].lower() == "sp" or fieldValue['field']['name'].lower() == "sprint points")):
                     sp = fieldValue.get('number')
-                    sp = '' if sp == None else int(sp)
+                    sp = int(sp) if sp != None else None
```

Comment:
Reverting this change, because to Kim's point, it is very easy to see which things need to be prioritized when poker says "None" instead of ""